_troubleshoot/index.md

```diff
@@ -116,3 +116,12 @@ The security plugin blocks the update by script operation (`POST <index>/_update
 ## Illegal reflective access operation in logs
 
 This is a known issue with Performance Analyzer that shouldn't affect functionality.
+
+
+## bootstrap checks failed: max virtual memory areas vm.max_map_count is too low
```

Comment:
We sentence-case headings, so "bootstrap" should be capitalized here. Also, my thinking is this heading is a bit long, but I'm open to suggestions here.